users/riblee/riblee.c

```diff
@@ -0,0 +1,312 @@
+/* Copyright 2020 Janos Daniel Reibl <janos.daniel.reibl@protonmail.com> @riblee
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "riblee.h"
+
+// Tap Dance definitions
+qk_tap_dance_action_t tap_dance_actions[] = {
+    [TD_A] = ACTION_TAP_DANCE_FN(dance_key_a),
+    [TD_E] = ACTION_TAP_DANCE_FN(dance_key_e),
+    [TD_I] = ACTION_TAP_DANCE_FN(dance_key_i),
+    [TD_O] = ACTION_TAP_DANCE_FN(dance_key_o),
+    [TD_U] = ACTION_TAP_DANCE_FN(dance_key_u),
+};
+
+// Tap Dance keys
+#define C_KC_A TD(TD_A)
+#define C_KC_E TD(TD_E)
+#define C_KC_I TD(TD_I)
+#define C_KC_O TD(TD_O)
+#define C_KC_U TD(TD_U)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
Throwing the `keymaps` object into userspace like this is kinda of a bad idea. It will cause issues if you then try and build for a different layout. Instead you should add to `layouts/community/ortho_5x12/riblee`, and rely on <https://docs.qmk.fm/#/feature_layouts> to pull the layout in.

Reply:
Thanks for the tip, I'll update the PR shortly.